### ISSUE TITLE
Document how to see validation issues while in development

### ISSUE THREAD

**Issue by sledorze** [twilio/guardrail#148]
Is there already a solution to this?

**Comment by sledorze** [comment]
Sorry, I mean the server validation of the Json models the endpoint is receiving.

If I send a payload which does not correspond to the Swagger model, I would like to know what's wrong.
May I lack some insight on how to retrieve that from the server implementation (using akka-http) but wondering if something implementation agnostic exists.

**Comment by blast-hardcheese** [comment]
Ah, this currently doesn't exist; I'm thinking it may be how we're doing the content validation exception, failing over to other available unmarshallers.

What kind of message would you expect to see? I'm assuming a Circe path would not be the best user experience; possibly a JSON path?

**Comment by sledorze** [comment]
@blast-hardcheese anything that actually provide an hint *and* supports custom validation messages.
I'm not sure about what looks like a circe-path or JSON path yet.. (I've left Scala since 2012 and am just back to it, landing in the ecosystem - looking at it ATM)

**Comment by sledorze** [comment]
@blast-hardcheese ok, I understand the problem now with circe-path.
I'm not sure but I guess the JSON Path would be extracted from the circe-path so this means we can have some way to handle directly a circe-path and possibly use a transformer to get a JSON path.
Exposing the circe DecodingFailure would be a good and the minimal, necessary first step.

**Comment by blast-hardcheese** [comment]
@sledorze I agree; I've made these changes in the akka-http generator for now.
